easybuild/tools/run.py

```diff
@@ -83,6 +83,10 @@
     "ulimit -u",  # used in det_parallelism
 )
 
+CWD_NOTFOUND_ERROR = (
```

Comment:
@lexming parking this in `tools.run` doesn't make much sense to me, I propose we move it to `tools.build_log`, see https://github.com/lexming/easybuild-framework/pull/4

If you agree with that minor change, this PR is good to merge

Reply:
`tools.build_log` makes more sense indeed, thx for the PR!